s2c_sensor_module/src/main.c

```diff
@@ -33,6 +33,10 @@
 // Function prototypes
 uint8_t get_pinstrap_id(void);
 inline float uint16ToC(uint16_t data);
+void flash_pinstrap_id(uint8_t pin_num);
```

Comment:
While there is some benefit in using the LED for these diagnostic/error reporting codes, I think it's too specific of a usage to be included in the base code. Let's maybe drop this commit from this pull request, but definitely good use of the LED for your purposes!